src/Microsoft.Framework.Runtime.Common/Impl/PlatformHelper.cs

```diff
@@ -2,18 +2,23 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.Framework.Runtime.Infrastructure;
 
 namespace Microsoft.Framework.Runtime
 {
     internal static class PlatformHelper
```

Comment:
Noooo, get rid of this and stop using CallContextServiceLocator everywhere :smile: